lerna.json

```diff
@@ -1,7 +1,7 @@
 {
   "npmClient": "yarn",
   "version": "independent",
-  "commands": {
+  "command": {
```

Comment:
Yikes, just discovered that the 3.0 lerna bump around this is volatile. Given https://github.com/lerna/lerna/tree/master/commands/publish#deprecated-options, we'll need to reconfigure to use the `version` command (instead of publish). This will also necessitate a change to this line: https://github.com/zendeskgarden/react-components/blob/master/utils/travis/publish-packages.js#L16

Reply:
See https://github.com/zendeskgarden/css-components/commit/7ba1e3648130089d762ed1552e88a1bfeb6e1dfa for details. Except note that the `"push": "false"` still isn't working as expected 😬

Reply:
...just tested and discovered that `"push": false` (no quotes) does the trick. Allowing you to check the commit/tags before sending off via `git push --follow-tags`. Oh, lerna.